controlpanel/api/models/policys3bucket.py

```diff
@@ -21,5 +18,16 @@ class Meta:
         unique_together = ("policy", "s3bucket")
         ordering = ("id",)
 
+    @property
```

Comment:
I'll probably go as far as renaming this to `arn` as we have other places and we call these properties `arn` so it may be good to have the same here for consistency.

Reply:
This property isn't actually used by anything any more, so I'm going to remove it.